ephios/core/services/notifications/types.py

```diff
@@ -581,14 +584,21 @@ def get_body(cls, notification):
     @classmethod
     def get_actions(cls, notification):
         event = Event.objects.get(pk=notification.data.get("event_id"))
+        participation = AbstractParticipation.objects.filter(
+            pk=notification.data.get("participation_id")
+        ).first()
```

Comment:
Why not?
```suggestion
        participation = AbstractParticipation.objects.get(pk=notification.data.get("participation_id"))
```
and 
```python
make_absolute(participation.participant.reverse_event_detail(event))
```
down below? 
If there's no participation we don't need to send anything...

Reply:
because this notification is also sent to all responsibles for the event. They don't have a participation for the event

Reply:
I see, can you please add a comment? 😅

Reply:
I mean the code that triggers that is only a few lines above, but sure ^^

Reply:
made me trip up